openslides_backend/action/actions/motion/follow_recommendation.py

```diff
@@ -47,12 +52,19 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
             lock_result=False,
         )
         recommendation_extension = instance.pop("recommendation_extension", None)
+        recommendation_extension_reference_ids = instance.pop(
+            "recommendation_extension_reference_ids", None
+        )
         if (
             recommendation_extension is not None
             and recommendation.get("show_state_extension_field")
             and recommendation.get("show_recommendation_extension_field")
         ):
             instance["state_extension"] = recommendation_extension
+            if recommendation_extension_reference_ids:
```

Comment:
if this is None, the content of `state_extension_reference_ids` remains unchanged and differs from it's `state_extension`. Is this intended?

Reply:
Good catch! Added a fix and a test for this case